GabeR

gabriellemrs16 opened the pull request:
- Issues I closed (#160, #171, #175)
- Please ignore anything pertaining to the reports (#169) --> I'm still working on that :)
- UI for order pages

abaporu-C commented:
![image](https://user-images.githubusercontent.com/77457350/158303090-cc3bb977-8e6c-4b39-ba4d-02e779f538ce.png)

This is what I see when I try to access Orders. I made a new migration and a new Database file, but it still doesn't work.

I noticed many changes to the Order model that I believe are the reasons for this issue. Many of the added features are things that are duplicated data from the Member foreign key (like HouseholdCode, HouseMember and the added Household foreign key that's unnecessary and is in conflict with the data model for the project already agreed upon).

Either way, I should be able to at least access your work on your branch when you have a pull request. Being unable to access what you've done from the get go is a big tell that your work is not done and you should work more on it before making a pull request.

If this branch is working on your end, please let me know and we can go over it in a call. I don't know why would work for you and not for me, but it is a possibility and I'm open to try to understand it.

Thanks for your work, but please deliver a working branch in your next request.

gabriellemrs16 commented:
It's because I forgot I added sorting and filtering. I fixed that error, but now I have a new one. I can always take out the sort/filter. Would we be able to call in the morning? Around 10?
